--- a/src/inkscape-version-info.cpp     2022-02-05 01:21:47.000000000 -0000
+++ b/src/inkscape-version-info.cpp     2022-04-15 16:20:31.762608261 -0000
@@ -21,7 +21,6 @@
 #include <gtkmm.h>
 #include <libxml2/libxml/xmlversion.h>
 #include <libxslt/xsltconfig.h>
-#include <poppler-config.h>

 #include "inkscape-version.h" // Inkscape version

@@ -91,7 +90,6 @@
     ss << "    Cairo version:    " << cairo_version_string() << std::endl;
     ss << "    Pango version:    " << pango_version_string() << std::endl;
     ss << "    HarfBuzz version: " << hb_version_string() << std::endl;
-    ss << "    Poppler version:  " << POPPLER_VERSION << std::endl;
     ss << std::endl;
     ss << "    OS version:       " << os_version();

@@ -110,4 +108,4 @@
   fill-column:99
   End:
 */
-// vim:filetype=cpp:expandtab:shiftwidth=4:tabstop=8:softtabstop=4:fileencoding=utf-8:textwidth=99:
\ No hay ningún carácter de nueva línea al final del fichero
+// vim:filetype=cpp:expandtab:shiftwidth=4:tabstop=8:softtabstop=4:fileencoding=utf-8:textwidth=99:
