--- a/src/style.cpp     2022-04-15 16:46:19.902493321 -0000
+++ b/src/style.cpp     2022-04-15 16:46:27.519238841 -0000
@@ -684,7 +684,6 @@
              * will fail, since CSS always overwrites SVG attributes.
              * Fixes Bug #324849
              */
-            g_warning_once("attribute 'clip-path' given as CSS");

             //XML Tree being directly used here.
             if (object) {
@@ -695,7 +694,6 @@
             /** \todo
              * See comment for SPAttr::CLIP_PATH
              */
-            g_warning_once("attribute 'mask' given as CSS");

             //XML Tree being directly used here.
             if (object) {
